src/components/MishnaView/UndividedText.js

```diff
@@ -3,7 +3,7 @@ import React from "react"
 import { clearPunctutationFromText, getTextForSynopsis, hideSourceFromText } from "../../inc/synopsisUtils"
 
 const UndividedText = props => {
-  const { lines, showPunctuation, showSources } = props
+  const { lines, showPunctuation, showSources, textAlignDirection } = props
```

Comment:
No need for textAlignDirection cause it will always be 'right'.

Reply:
> 
@rugggger 
removed the prop and set it to always right